README.md

```diff
@@ -59,7 +59,7 @@ This section explains the individual steps to retrieve the necessary files and b
 
 Steps:
 
- * Create a directory to hold your application e.g. (/home/<user>/aws_iot/my_app)
+ * Create a directory to hold your application e.g. (/home/&lt;user&gt;/aws_iot/my_app)
```

Comment:
Would putting in a backslash before the brackets work? i.e., `/home/\<user>/aws_iot/my_app` => /home/\<user>/aws_iot/my_app